common/test/acceptance/tests/lms/test_lms_user_preview.py

```diff
@@ -234,7 +234,6 @@ def test_student_state_delete_for_problem_loaded_via_ajax(self):
                          'for user {}'.format(self.USERNAME), msg)
 
 
-@attr(shard=3)
```

Comment:
Moving this to individual test points so that the a11y test point included in this class does not also run in the bok choy phase.